browser/ui/brave_browser_command_controller.cc

```diff
@@ -294,6 +294,13 @@ void BraveBrowserCommandController::InitBraveCommandState() {
 
   UpdateCommandEnabled(IDC_TOGGLE_ALL_BOOKMARKS_BUTTON_VISIBILITY, true);
 
+  if (browser_->is_type_normal()) {
+    // Delete these when upstream enables by default.
```

Comment:
I assume that upstream will enable them by default in cr127 as these are in the condition of 2023Refresh.